pom.xml

```diff
@@ -70,6 +71,18 @@
             <artifactId>display-url-api</artifactId>
             <version>2.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>okhttp</artifactId>
+            <version>${okhttp3.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>okhttp-urlconnection</artifactId>
+            <version>${okhttp3.version}</version>
+        </dependency>
+
```

Comment:
Afaict you need to add a dependency for `okio`

Reply:
@jsoref 
Thanks, I'll give that a try!

Reply:
@jsoref  Done.  Please try it now.